### DISCUSSION TITLE
When hover the cursor over the record, only this record would increase to the settings Comfortable from Compact

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#10600] [Feature Requests]
When you add items from another collection to the collection, the list of items appears on the right side - an interesting solution would be if you hover the cursor over the record, this record would increase to the settings Comfortable from Compact (only this one record). It would be easier to see a photo. Maybe otherwise in the column with photos, after hovering over the photo, the photo would appear enlarged in the form of a pop-up window, as it is in the status.

**Comment by rijkvanzanten** [comment]
Interesting idea! I'm wondering if the constantly jumping rows would be annoying to use in real life 🤔

**Reply by koksikus** [reply]
Thanks for the answer. You may be right it can be annoying. Perhaps some animation would help, it wouldn't hurt the eyes so much. Most people reduce the view in the browser to display more data, but the photos are no longer legible and to view you need to enter the record. Maybe an interesting solution would be that when you hover over the data column, they would appear as it happens in the status. Maybe in the settings, you could mark which data can be displayed this way.

**Reply by rijkvanzanten** [reply]
Another interesting approach could be to add a hover state to the image display itself that shows the image bigger in a little tooltip, rather than making the whole row bigger 🤔

**Reply by benhaynes** [reply]
I like that approach, Rijk...  Ties into our other conversations about display hover/click.

**Reply by koksikus** [reply]
Or maybe after the cursor stop for 5 seconds on the record, then it would change with the animation to Comfortable. This is just an extension of the idea. I also think it's better if "add a hover state to the image display itself that shows the image bigger in a little tooltip".
This can be the case for each field, only in the settings you choose for each field whether you want the content to be displayed in this way.

Thanks gentlemen - Directus rules!!!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
